Update for pre-plan run tasks beta (#72)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx

The POST request submits a JSON object with the following properties as a request payload.

#### Common Properties

All request payloads contain the following properties.

|            Key path             |  Type   |          Values          |                                                 Description                                                 |
| ------------------------------- | ------- | ------------------------ | ----------------------------------------------------------------------------------------------------------- |
| `payload_version`               | integer | `1`                      | Schema version of the payload. Only `1` is supported.                                                       |
| `access_token`                  | string  |                          | Bearer token to use when calling back to Terraform Cloud.                                                   |

> | `stage`                         | string  | `pre_plan`, `post_plan`, `pre_apply` | The [stage](/cloud-docs/run/states) the task was triggered at.                                              |

Reviewer: You mentioned this is in beta right? I forgot that during my first pass! Also forgive me if I'm adding an incorrect actor again. I just hate ending on "at" - let me know if this works!

```suggestion
| `stage`                         | string  | `pre_plan`, `post_plan`, `pre_apply` | The [run stage](/cloud-docs/run/states) when Terraform Cloud triggers the run task.  The `stage` property is in beta.                                            |
```

> | `stage`                         | string  | `pre_plan`, `post_plan`, `pre_apply` | The [stage](/cloud-docs/run/states) the task was triggered at.                                              |

Author: It's still _possible_ for an old pre-apply task to exist and emit a payload. So it's _technically_ possible, but highly unlikely right now.

| `vcs_commit_url`                | string  |                          | URL to the commit that triggered this run. This is `null` if the workspace does not a VCS repository.               |

#### Pre-Plan Properties

-> **Note:** Pre-plan run tasks are in beta.

Requests with `stage` set to `pre_plan` contain the following additional properties.

| Key path                             | Type   | Values | Description                                                                                                                                                                                      |
|--------------------------------------|--------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `configuration_version_id`           | string |        | The ID of the [configuration version](/cloud-docs/api-docs/configuration-versions) for the run.                                                                                                      |

> | `configuration_version_download_url` | string |        | The URL to [download the configuration version](cloud-docs/api-docs/configuration-versions#download-configuration-files). This is `null` if the configuration version cannot be downloaded. |

Reviewer: ```suggestion
| `configuration_version_download_url` | string |        | The URL to [download the configuration version](cloud-docs/api-docs/configuration-versions#download-configuration-files). This is `null` if the configuration version is not available to download. |
```
